maven/bnd-export-maven-plugin/README.md

```diff
@@ -26,3 +27,16 @@ a runnable jar into the targetDir. If not specified targetDir defaults to the pr
             </plugin>
 ```
 
+## Configuration Properties
+
+|Configuration Property | Description |
+| ---                   | ---         |
+|`bndruns`              | Contains at least one `bndrun` child element, each element naming a bndrun file defining a runtime and tests to execute against it.|
+|`targetDir`            | The director into which to export the result. _Defaults to `${project.build.directory}`._|
+|`resolve`              | Whether to resolve the `-runbundles` required for a valid runtime. _Defaults to `false`._|
+|`failOnChanges`        | Whether to fail the build if any change in the resolved `-runbundles` is discovered. _Defaults to `true`._|
+
+## Breaking Change (as of 3.4)
```

Comment:
I would call this out in https://github.com/bndtools/bnd/wiki/Changes-in-3.4.0 rather than in this readme.